macros/models/_project/balancer/balancer_liquidity_macro.sql

```diff
@@ -58,11 +58,12 @@ WITH pool_labels AS (
         SELECT 
             day,
             contract_address AS token,
+            decimals,
             bpt_price
         FROM {{ ref('balancer_bpt_prices') }}
         WHERE blockchain = '{{blockchain}}'
         AND version = '{{version}}'
-        GROUP BY 1, 2, 3
+        GROUP BY 1, 2, 3, 4
```

Comment:
If we group by all columns, maybe we can use `DISTINCT` here?

Reply:
fair enough. Thanks for the review!